facebook-commerce.php

```diff
@@ -2746,14 +2743,18 @@ public function is_legacy_feed_file_generation_enabled() {
 	 * Determines whether the Facebook messenger is enabled.
 	 *
 	 * @since 1.10.0
+	 * @deprecated 3.2.0
 	 *
 	 * @return bool
 	 */
 	public function is_messenger_enabled() {
+		return false;
+
 		/**
 		 * Filters whether the Facebook messenger is enabled.
 		 *
 		 * @since 1.10.0
+		 * @deprecated 3.2.0
```

Comment:
nit: since the filter is never run and we return false. I think we can remove it.